scripts/stats/wafs/exevs_wafs_atmos_stats.sh

```diff
@@ -8,9 +8,7 @@
 set -x
 
 cd $DATA
-if [ -s wafs_stat.cmdfile ];then
-	rm wafs_stat.cmdfile
-fi
+rm -f wafs_stat.cmdfile
 
```

Comment:
@YaliMao-NOAA Why are we removing the if block here?  If the wafs_stat.cmdfile doesn't exist, then line 11 here will cause a "No such file" warning,  wouldn't it?

Reply:
Oh, the -f prevents that No such file message from happening.  I stand corrected.